### ISSUE TITLE
create-stub not working with osx monterey

### ISSUE THREAD

**Issue by ansonlouis** [flow-typed/flow-typed#4195]
### CLI Version

3.5.0

### Operating system

macOS Monterey 12.x

### Bug description

I'm getting a JS error from a deep dependency being outdated while trying to run `create-stub` on monterey.

```
TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
    at macosRelease (/<path>/node_modules/macos-release/index.js:26:26)
    at osName (/<path>/node_modules/os-name/index.js:21:18)
    at getUserAgent (/<path>/node_modules/@octokit/rest/node_modules/universal-user-agent/dist-node/index.js:11:53)
    at parseOptions (/<path>/node_modules/@octokit/rest/lib/parse-client-options.js:80:57)
    at new Octokit (/<path>/node_modules/@octokit/rest/lib/constructor.js:23:31)
    at Object.<anonymous> (/<path>/node_modules/flow-typed/dist/lib/github.js:10:16)
```

Seems like the Octokit version you use is somewhat outdated. They have some dependencies that eventually lead to the following block of code in `macos-release` package.

```
const nameMap = new Map([
	[20, ['Big Sur', '11']],
	[19, ['Catalina', '10.15']],
	[18, ['Mojave', '10.14']],
        ...
```
The newer versions of Octokit seem to not even have those deps anymore, so updating should fix this issue I assume (but don't know if it will cause any new issues for you 😄)

### Steps to reproduce

Run `create-stub` for some package while on osx monterey

### Flow configuration

n/a

### Do you want to submit a pull request to fix this bug?

No (hope someone else can)

**Comment by Brianzchen** [comment]
Thanks for raising this, should be easy to upgrade this dep is only used for running out test suite to find flow released versions to run tests against.

**Comment by Brianzchen** [comment]
@ansonlouis, we've just released [3.6.0](https://github.com/flow-typed/flow-typed/releases/tag/v3.6.0), let us know you come across anymore issues

**Comment by ansonlouis** [comment]
Thanks!
